frontend/src/scenes/session-recordings/player/controller/PlayerController.tsx

```diff
@@ -122,7 +162,10 @@ export function PlayerController(): JSX.Element {
                     {!isCinemaMode && playerMode === SessionRecordingPlayerMode.Standard && (
                         <>
                             <CommentOnRecordingButton />
+                            <Screenshot />
+                            {featureFlags[FEATURE_FLAGS.REPLAY_EXPORT_SHORT_VIDEO] && <Clip />}
                             {playlistLogic ? <PlayerUpNext playlistLogic={playlistLogic} /> : undefined}
+                            <span className="text-muted text-xs px-1">|</span>
```

Comment:
there's `LemonDivider` for this use case

but because we use the vertical line for side actions just by here with the comment button
this vertical line threw me and I think we can live without it